### PR TITLE
ref(consent): Add helptext to Terms.

### PR DESCRIPTION
**Pull request by lauryndbrown**
Adds a help text to the terms, so that not everything is written in the label.

Fixes: https://github.com/carpentries/amy/issues/1861

### CODE REVIEW

**Comment by lauryndbrown** [comment]
Applied dropdown changes on the Actions required page

Screen Shot 2021-03-24 at 12 47 41 PM

**Comment by lauryndbrown** [comment]
Screen Shot 2021-04-05 at 3 03 31 PM

File: amy/consents/migrations/0002_term_help_text.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.17 on 2021-04-05 19:56
```
Comment (pbanaszkiewicz) [inline amy/consents/migrations/0002_term_help_text.py:1]: I think you have another `0002` migration in #1880.

File: amy/workshops/management/commands/fake_database.py [hunk 2]
```diff
@@ -894,12 +895,12 @@ def fake_terms_and_consents(self):
...
         )
         public_profile = Term.objects.create(
-            content="Consent to making profile public:"
-            " Allow to post your name and any public profile"
-            " you list (website, Twitter) on our instructors website."
-            " Emails will not be posted.",
+            content="Consent to making profile public",
             required_type=Term.PROFILE_REQUIRE_TYPE,
             slug="public-profile",
+            help_text=" Allow to post your name and any public profile"
```
Comment (pbanaszkiewicz) [inline amy/workshops/management/commands/fake_database.py:901]: Space before `Allow`.

**Review by pbanaszkiewicz — APPROVED** [review]

File: amy/consents/migrations/0002_term_help_text.py (hunk 1 above)
Comment (lauryndbrown) [reply amy/consents/migrations/0002_term_help_text.py:1]: Good catch. Thanks!

**Pull request merged**